.gitignore

```diff
@@ -1 +1,3 @@
 node_modules/
+*npm-debug*
+*yarn-error*
```

Comment:
@lumodon they are most likely unnecessary, but theoretically I believe they will add anything with npm-debug or yarn-error in it. Like, folder/npm-debug.someextentsion, or, yarn-error.lock or .log or anything else yarn might throw at me.